src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -53,13 +53,16 @@ public class Intake extends SubsystemBase {
 
   // local variables
   private double m_setPoint;
-  private double m_IntakePiviotVoltage = 0.0;
+
+  private double intakePivotVoltage = 0.0;
+  private double intakeSpeedVoltage = 12.0;
+  private double intakeRestVoltage = 0.0;
 
   // WPILib class objects
-  private CANSparkMax intakeMotor;
+  private CANSparkMax m_IntakeMotor;
```

Comment:
if we want to use `m_` we should use it on all member variables, as in all variables visible to an entire class. That is what` m_` means, member. IDC if we use it or not but we should be consistent.